advanced/Scripts/query.sh

```diff
@@ -133,7 +133,7 @@ scanDatabaseTable "${domainQuery}" "whitelist" "${exact}"
 scanDatabaseTable "${domainQuery}" "blacklist" "${exact}"
 
 # Scan Regex table
-mapfile -t regexList < <(sqlite3 "${gravityDBfile}" "SELECT domain FROM vw_regex" 2> /dev/null)
+mapfile -t regexList < <(sqlite3 "${gravityDBfile}" "SELECT domain FROM vw_regex_blacklist" 2> /dev/null)
```

Comment:
Are we adding regex whitelist support to this file in this PR or later?

Reply:
Thanks for reminding me, I added whitelist regex support for `pihole -q`